scene/3d/audio_stream_player_3d.h

```diff
@@ -80,17 +80,17 @@ class AudioStreamPlayer3D : public Node3D {
 	uint64_t last_mix_count = -1;
 	bool force_update_panning = false;
 
-	static void _calc_output_vol(const Vector3 &source_dir, real_t tightness, Vector<AudioFrame> &output);
+	static void _calc_output_vol(const Vector3 &source_dir, real_t tightness, LocalVector<AudioFrame> &output);
 
-	void _calc_reverb_vol(Area3D *area, Vector3 listener_area_pos, Vector<AudioFrame> direct_path_vol, Vector<AudioFrame> &reverb_vol);
+	void _calc_reverb_vol(Area3D *area, Vector3 listener_area_pos, LocalVector<AudioFrame> direct_path_vol, LocalVector<AudioFrame> &reverb_vol);
```

Comment:
```suggestion
	void _calc_reverb_vol(Area3D *area, Vector3 listener_area_pos, const LocalVector<AudioFrame> &direct_path_vol, LocalVector<AudioFrame> &reverb_vol);
```
Or you'll do copy every single time it's called

Reply:
Thanks!